openmdao/core/tests/test_distrib_derivs.py

```diff
@@ -588,6 +588,49 @@ def test_distribcomp_derivs_feature(self):
         assert_near_equal(J[('C2.outvec', 'indep.x')], 
                           np.eye(15)*np.append(2*np.ones(8), -3*np.ones(7)))
 
+@unittest.skipUnless(MPI and PETScVector, "MPI and PETSc are required.")
+class ZeroLengthInputsOutputs(unittest.TestCase):
+
+    N_PROCS = 4
+    # this test case targets situations when zero-length inputs
+    # or outputs are located on some processors
+    # issue 1350
+
+    def test_distribcomp_zerolengthinputsoutputs(self):
+        import numpy as np
```

Comment:
Don't need to put imports here unless this test is going to show up in the docs.